TipCalculator/ViewController.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  ViewController.swift
+//  TipCalculator
+//
+//  Created by Michail on 9/12/16.
+//  Copyright © 2016 mishaparadising. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    // MARK: Outlets
+    @IBOutlet weak var sumCountField: UITextField!
+    @IBOutlet weak var tipTitleLabel: UILabel!
+    @IBOutlet weak var tipValueLabel: UILabel!
+    @IBOutlet weak var totalValueLabel: UILabel!
+    
+    @IBOutlet weak var tipSlider: UISlider!
+    
+    // MARK: Properties
+    var sum = Sum(current: 0, tipPercentage: 15)
+    
+    // MARK: Init
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        initTipSlider()
+        addToolbarToNumberPad()
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    func initTipSlider() {
+        tipSlider.maximumValue = 100
+        tipSlider.minimumValue = 0
+        
+        tipSlider.setValue(Float(15), animated: false)
+        
+        tipSlider.userInteractionEnabled = false
+    }
+    
+    // MARK: Actions
+    @IBAction func tipSliderAction(sender: UISlider) {
+        tipTitleLabel.text = "Tip(\(Int(sender.value))%):"
+        sum.tipPercentage = Double(sender.value);
```

Comment:
No need to use `Double` anywhere in this class, convert to `Float` **everywhere**.